test/sdk/e2e.js

```diff
@@ -7,6 +7,8 @@ const expect = chai.expect;
 chai.use(chaiAsPromised);
 require('dotenv').config({ path: `${process.cwd()}/test/sdk/.env` });
 
+const ACCEPTANCE_CHAIN_ID = 80001;
+const ACCEPTANCE_NETWORK = 'Mumbai';
 describe('Original sdk e2e-method tests', async () => {
```

Comment:
Didn't feel this needed to be in secrets, as it's public knowledge and wont be changing. 
@don1989 wdyt?

Reply:
Agreed